concept/type/ThingType.java

```diff
@@ -51,17 +54,27 @@
 
     Stream<? extends AttributeType> getOwns();
 
+    Stream<? extends AttributeType> getOwnsDirect();
+
     Stream<? extends AttributeType> getOwns(boolean onlyKey);
 
+    Stream<? extends AttributeType> getOwnsDirect(boolean onlyKey);
+
     Stream<? extends AttributeType> getOwns(AttributeType.ValueType valueType);
 
+    Stream<? extends AttributeType> getOwnsDirect(AttributeType.ValueType valueType);
+
     Stream<? extends AttributeType> getOwns(AttributeType.ValueType valueType, boolean onlyKey);
 
+    Stream<? extends AttributeType> getOwnsDirect(AttributeType.ValueType valueType, boolean onlyKey);
```

Comment:
Are all of these overloads used? I think it's enough to just implement the ones that we use for the exporter

Reply:
Emm...I think from an API point of view, it's better to make the complete. If we provide an API for `getOwns` then there should be an API for `getOwnsDirect`?

Reply:
Yeah that's fine as well.